tests/test_lockfile.py

```diff
@@ -0,0 +1,24 @@
+# Copyright FuseSoC contributors
+# Licensed under the 2-Clause BSD License, see LICENSE for details.
+# SPDX-License-Identifier: BSD-2-Clause
+
+import pytest
+
+
+def test_load_lockfile():
+    import os
+
+    from fusesoc.lockfile import load_lockfile
+    from fusesoc.vlnv import Vlnv
+
+    lockfile_dir = os.path.join(os.path.dirname(__file__), "lockfiles")
+    lockfile = load_lockfile(os.path.join(lockfile_dir, "works.lock"))
```

Comment:
```suggestion
    lockfile_dir = Path(__file__).parent / "lockfiles"
    lockfile = load_lockfile(lockfile_dir / "works.lock")
```

Now that `load_lockfile` takes a path and not a string. We should probably be using `pathlib.Path` here.

Reply:
Agree. We want to move evertything to pathlib eventually, so new code should definitely use it.

Reply:
Fixed.